### DISCUSSION TITLE
Question about the Update Banner at top of screen

### DISCUSSION THREAD

**Discussion by GlydetheSurfer** [AdguardTeam/AdGuardHome#6580] [Q&A] [answered]
I have a travel router from GL.iNet that i have adguard home installed on that i use to block ads with. but the firmware hasnt been updated in forever and there is no other update for it. and since it has no update, adguard home cant ever be updated on it anymore. so since this is the case... adguard home keeps checking for updates and always display this stupid "adguard home [insert version number here] is now available" banner at the top of the screen. since i can no longer update adguard home on this router, i at least want to get rid of this stupid banner.. is there any way to do that?
![image](https://github.com/AdguardTeam/AdGuardHome/assets/32045132/b6676b48-a43f-47b0-9bbc-f985c64e4992)

**Comment by ainar-g** [comment, marked as answer]
Not unless you can get to the service configuration files.  If you can, `--no-check-update` flag should help.
